source/system-administrators/activities/clustering/changing-git-url-format-in-cluster.rst

```diff
@@ -0,0 +1,87 @@
+:is-up-to-date: True
+
+.. index:: Changing the Cluster Git URL, Studio Clustering, Clustering
+
+.. _changing-the-cluster-git-url:
+
+============================
+Changing the Cluster Git URL
+============================
+
+When the cluster Git URL for syncing members is changed after a cluster has been setup and started, the nodes on the disk may contain the old URL format when starting up and there may be an error in the logs like below:
+
+   .. code-block:: text
+
+      [ERROR] 2021-03-12T18:54:02,887 [pool-5-thread-10] [job.StudioClockExecutor] | Error executing Studio Clock Job
+      java.lang.ClassCastException: org.eclipse.jgit.transport.TransportHttp cannot be cast to org.eclipse.jgit.transport.SshTransport
```

Comment:
This error only occurs when you change the URL from SSH to HTTPS. Maybe we need to clarify that?